database/src/main/resources/db/migration/V1.46.0__endpoint_stauts.sql

```diff
@@ -0,0 +1,7 @@
+-- add a new status column
+
+alter table endpoints add column status integer ;
+
+update endpoints set status = 0 where status is null;
+
+create index status_idx ON endpoints (status);
```

Comment:
I have two questions about this index:
- Should it be partial? I would say yes.
```suggestion
create index status_idx ON endpoints (status) where status not in (0, 5);
```
- Should it allow index-only scans? That's probably overkill.
```suggestion
create index status_idx ON endpoints (status, id, account_id, endpoint_type, endpoint_sub_type, enabled, name, description, created, updated) where status not in (0, 5);
```